podcasts/DiscoverSummaryProtocol.swift

```diff
@@ -6,7 +6,7 @@ protocol DiscoverSummaryProtocol: AnyObject {
     func populateFrom(item: DiscoverItem)
 }
 
-protocol DiscoverDelegate: AnyObject {
+protocol DiscoverDelegate: UIViewController {
```

Comment:
I think I did this because passing the delegate into `` was complaining about the type. But that should have been a clue I was doing it wrong.

The new delegate methods you suggested fix this up.